gcp/table_gcp_compute_instance.go

```diff
@@ -293,6 +293,7 @@ func listComputeInstances(ctx context.Context, d *plugin.QueryData, h *plugin.Hy
 	}
 
 	filterQuals := []filterQualMap{
+		{"name", "name", "string"},
```

Comment:
@JoshRosen Similar to this change, do we also need this change in the `gcp_compute_disk` table's `listComputeDisk` function?

Reply:
Good catch; updated.